lxd/device/tpm.go

```diff
@@ -165,12 +167,22 @@ func (d *tpm) startContainer() (*deviceConfig.RunConfig, error) {
 		return nil, fmt.Errorf("Failed to get TPM device information")
 	}
 
```

Comment:
What are these numbers? Can we define them as constants? Or comment to explain what the logic is doing here for future readers?